src/dbnode/client/write_test.go

```diff
@@ -169,13 +233,30 @@ func setShardStates(t *testing.T, s *session, host topology.Host, state shard.St
 	}
 }
 
+func replaceHost(t *testing.T, s *session, leavingHost topology.Host, initializingHost topology.Host) {
```

Comment:
nit: `markHostReplacement` is a little clearer than `replaceHost` (the latter is implying you're doing a replace operation, which you're not doing)